api-admin/openapi.yaml

```
@@ -24,6 +24,19 @@ paths:
         500:
           description: Something goes wrong
 
+  "/users.get":
+    get:
+      operationId: usersGetRegisteredUsers
```

Имя операции это не дескрипшен, это имя используется для генерации имени эффекта в жс коде и в раст коде.

не оч удобно будет такое длинное имя дергать, мне кажется.

```suggestion
      operationId: usersGetRegisteredUsers
```